evap/static/ts/src/csrf-utils.ts

```diff
@@ -1,35 +0,0 @@
-// based on: https://docs.djangoproject.com/en/3.1/ref/csrf/#ajax
```

Comment:
Why did you delete this file / move it into `utils`? The `csrftoken` etc. could very well stay here